### PR TITLE
Deploy to govcloud

### PR DESCRIPTION
**Pull request by jcscottiii**
Fixes issue(s) # https://github.com/18F/cg-migration/issues/37 Migrate to GovCloud.

Changes proposed in this pull request:

- Use two sets of deployer credentials. one for prod and one for staging via deployer account broker
- update cf cli 
- update manifest to use whole domain `dolores-app.18f.gov` vs the domain `18f.gov` with hostname `dolores-app`

/cc cc @fureigh @jessieay

### CODE REVIEW

**Comment by jessieay** [comment]
Can you update the deployment instructions in `CONTRIBUTING` ? https://github.com/18F/dolores-landingham-slack-bot/blob/develop/CONTRIBUTING.md#manual-deployment

**Comment by jcscottiii** [comment]
@jessieay done!

File: manifest_base.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# base configuration
+---
+command: script/start
+memory: 1GB
+buildpack: https://github.com/cloudfoundry/ruby-buildpack.git
```
Comment (jmcarp) [inline manifest_base.yml]: Could we use `ruby_buildpack` instead?

Comment (jcscottiii) [inline manifest_base.yml]: done!

**Comment by jcscottiii** [comment]
@jessieay @jmcarp Feel free to merge!

**Pull request merged**